src/toil/common.py

```diff
@@ -51,7 +51,7 @@
 from toil import lookupEnvVar
 from toil.version import dockerRegistry, dockerTag
 
-
+# aim to pack autoscaling jobs within a 20 minute block before provisioning a new node
```

Comment:
Sorry, this is such a nitpick, but I gotta point out that this is 30 minutes, not 20.

Reply:
Sorry, I had it at 1200, but the autoscaling tests failed (possibly still flaky, but I think this may have actually caused them to fail).  Anyway, I forgot to set it back, thanks for catching this!